index.html

```diff
@@ -1526,35 +1526,40 @@ <h3 id="lifetime-AudioContext" class="informative">
         </section>
         <section>
           <h3>
-            System resources associated with an AudioContext
+            System resources associated with BaseAudioContext subclasses
           </h3>
           <p>
-            <a>AudioContext</a>s should be considered expensive objects.
-            Creating an <a>AudioContext</a> often involves creating a
-            high-priority thread, and using a low-latency system audio stream,
-            both having an impact on energy consumption. Creating more than one
-            <a>AudioContext</a> in a document is most of the time unnecessary.
+            The subclasses <a>AudioContext</a> and <a>OfflineAudioContext</a>
+            should be considered expensive objects. Creating these objects may
+            involve creating a high-priority thread, and using a low-latency
+            system audio stream, both having an impact on energy consumption.
+            It is usually not necessary to create more than one
+            <a>AudioContext</a> in a document.
+          </p>
+          <p>
+            Constructing or resuming a <a>BaseAudioContext</a> subclass
+            involves <dfn id="acquiring">acquiring system resources</dfn>. This
+            means creating a high-priority thread and (for <a>AudioContext</a>
+            only) creating a system audio stream. This operation returns when
+            the audio stream is ready.
```

Comment:
Do we have a concept of "ready" really means?  If so, a link to this would be great.